src/jwplayer/utils/globalJWInterface.ts

```diff
@@ -6,97 +6,97 @@
 import { Player } from 'jwplayer/types';
 
 export interface WindowWithJWPlayer {
-	jwplayer: () => Player;
+	jwplayer: (playerId: string | null) => Player;
```

Comment:
is the implementation of this function somewhere else?

Reply:
Yes, it's an external code. Created by 3rd-party JWPlayer: https://docs.jwplayer.com/players/reference/javascript-player-api-introduction